shub/image/build.py

```diff
@@ -53,15 +57,20 @@ def build_cmd(target, version, skip_tests):
     if not os.path.exists(os.path.join(project_dir, 'Dockerfile')):
         raise shub_exceptions.BadParameterException(
             "Dockerfile is not found, please use shub image 'init' command")
-    is_built = False
+    bar, is_built = None, False
+    verbose = utils.is_verbose()
     for data in client.build(path=project_dir, tag=image_name, decode=True):
         if 'stream' in data:
+            if not verbose:
+                bar = _create_or_update_progress_bar(bar, data, verbose)
             utils.debug_log("{}".format(data['stream'][:-1]))
             is_built = re.search(
                 r'Successfully built ([0-9a-f]+)', data['stream'])
         elif 'error' in data:
-            click.echo("Error {}:\n{}".format(
+            tqdm.write("Error {}:\n{}".format(
```

Comment:
Yes, and sometimes it leads to bar duplication, with `click.echo`:
```
layers:   0%|                                                                                                                                            | 0/9Error The command '/bin/sh -c abcdef' returned a non-zero code: 127:
{'code': 127, 'message': "The command '/bin/sh -c abcdef' returned a non-zero code: 127"}
layers: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 9/9
Error: Build image operation failed
```
With `tqdm.write`:
```
Error The command '/bin/sh -c abcdef' returned a non-zero code: 127:
{'code': 127, 'message': "The command '/bin/sh -c abcdef' returned a non-zero code: 127"}
layers: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 9/9
Error: Build image operation failed
```
[tqdm.write](https://github.com/tqdm/tqdm#writing-messages) respects existing tqdm progress bars and updates it properly.